db/schema.rb

```diff
@@ -10,20 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171004033158) do
+ActiveRecord::Schema.define(version: 20171013145239) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "games", force: :cascade do |t|
-    t.integer  "white_player_id"
```

Comment:
The only thing I did was add the color column. Not sure why these got deleted - they're not even in the same table...

Reply:
Was the migration that adds these columns not checked in?

Make sure it's in master and also in this branch.